pkg/push/push.proto

```diff
@@ -46,4 +46,8 @@ message EntryAdapter {
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "structuredMetadata,omitempty"
   ];
+  repeated LabelPairAdapter parsed = 4 [
+    (gogoproto.nullable) = false,
+    (gogoproto.jsontag) = "parsed,omitempty"
+  ];
```

Comment:
I would love to avoid doing this change since it adds an unexpected field to the push payload. I know it is hard but maybe leave a todo and let us try fixing it in a separate PR to avoid overcomplicating this one.

Reply:
> I would love to avoid doing this chang

Likewise! The problem is this is the object we move around everywhere an throughout our components so we need to send it around somewhere.

> it adds an unexpected field to the push payload

I don't think that's a problem. Protobuf is extensible and backwards compatible when adding fields to the end of an object definition. So in this case, [it's still backwards compatible](https://protobuf.dev/programming-guides/proto3/#updating).

Reply:
I am not worried about backwards compatibility, my ony concern is having an unwanted field on the proto definitions on the write path. I understand it is hard to do it so maybe leave a todo for now.